pkg/controller/noderefutil/util.go

```diff
@@ -60,7 +60,7 @@ func GetReadyCondition(status *corev1.NodeStatus) *corev1.NodeCondition {
 
 // IsNodeReady returns true if a node is ready; false otherwise.
 func IsNodeReady(node *corev1.Node) bool {
-	if node == nil || &node.Status == nil {
+	if node == nil {
```

Comment:
This was reported by govet as `nilness: impossible condition: non-nil == nil (govet)` which makes sense given that it's referencing a non-pointer field, which can never be nil.